File: tests/conformance/fixtures/typescript/10-task-inputs.ts
import { Pipeline } from '../../../../sdk/typescript/src/index';

const p = new Pipeline();

p.task('build').run('go build -o /out/app')
  .output('binary', '/out/app');

p.task('test').run('./app test')
  .inputFrom('build', 'binary', '/app');

p.emit();

File: tests/conformance/fixtures/elixir/10-task-inputs.exs
use Sykli

pipeline do
  task "build" do
    run "go build -o /out/app"
    output "binary", "/out/app"
  end

  task "test" do
    run "./app test"
    input_from "build", "binary", "/app"
  end
end
|> Sykli.Emitter.validate!()
|> Sykli.Emitter.to_json()
|> IO.puts()
